assets/src/stories-editor/blocks/amp-story-page/copy-paste-handler.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { pasteHandler, serialize } from '@wordpress/blocks';
+import { documentHasSelection } from '@wordpress/dom';
+import { withDispatch, useSelect, useDispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { ensureAllowedBlocksOnPaste } from '../../helpers';
+
+function CopyPasteHandler( { children, onCopy, clientId, isSelected } ) {
+	const {
+		isFirstPage,
+		canUserUseUnfilteredHTML,
+	} = useSelect(
```

Comment:
Should `useSelect` be used? We cannot reliably use it just yet. See #2538 (comment)

Reply:
We can now use `useSelect` since #3167, see https://github.com/ampproject/amp-wp/issues/2538#issuecomment-527228741. Or do you mean because of the issue with scalar values?

Reply:
Yeah, we can use `useSelect`